selfdrive/manager.py

```diff
@@ -505,9 +505,6 @@ def manager_thread():
     # with 2% left, we killall, otherwise the phone will take a long time to boot
     should_start = should_start and avail > 0.02
 
-    # require usb power
```

Comment:
somewhat a problem is that in chffr+ (passive==0) you'll have `should_start` becoming true just by walking around with the Eon in hands.